OracleWebCenterSites/README.md

```diff
@@ -0,0 +1,430 @@
+   | --v                    | user_projects_volume_dir| ‘/scratch/DockerVolume/WCSitesVolume/WCSites’ mounts the host directory as a Volume.      |
+   | --v                    | sites_shared_volume_dir | ‘/scratch/DockerVolume/WCSitesVolume/WCSitesShared’ mounts the host directory as a Volume.|
+   | --env-file             | environment_file        | ‘ wcsitesadminserver.env.list’ sets the environment variables.                            |
+   | oracle/wcsites:12.2.1.3| repo_name:tag_name      | Repository name, Tag name of the image.                                                   |
+```
+**IMPORTANT**: Monitor the container logs to check if the WebLogic server starts up before logging in to the Console.
+
+For monitoring Docker container Logs:
+```
+    $ docker logs -f --tail 900 WCSitesAdminContainer
+```
+**Note**: Copy the **WebLogic Admin** and **Database Schema** passwords from the log. It's used while starting the Managed container.
+
+To connect to the container for monitoring WebCenter Sites/WebLogic Logs:
+```
+   $ docker exec -it WCSitesAdminContainer /bin/bash
+```
+Now you can access below WebLogic Console.
+```
+   http://DOCKER_HOST:7001/console 
+```
+### B. Creating a managed container for WebCenter Sites Managed Server 
+This container is used to manage the Managed Server. 
+
+#### 1. Create the environment file to passed the parameters 
+Update the environment `wcsitesserver.env.list` file which is located at `../docker-images-master/OracleWebCenterSites/dockerfiles/`.
+
+`wcsitesserver.env.list` file details:
+```
+   WCSITES_ADMIN_HOSTNAME=<WCSites Admin Container Name>
+   DOMAIN_NAME=<Domain Name>
+   SITES_SERVER_NAME=<Sites Server Name>
+   ADMINUSERNAME=<Admin Username>
+   ADMINPASSWORD=<Admin Password (use the password that was copied earlier)>
+```
+#### 2. Start the Managed Container
+
+a. To run WebLogic Managed Server container, go to `dockerfiles` folder located at `../docker-images-master/OracleWebCenterSites/`.  
+ 
+b. Run the following command and pass the environment file name as a parameter: 
+```
+   $ docker run -d -t --name <container_name> --network=<network_name> --volumes-from <admin_container_name> -p <sites_port>:7002 -p <sites_ssl_port>:9002 --env-file <environment_file> <repo_name:tag_name> /bin/bash -c "/u01/oracle/sites-container-scripts/startSitesServer.sh; /bin/bash"
+```
+Sample command:
+```
+   $ docker run -d -t --name WCSitesManagedContainer --network=WCSitesNet --volumes-from WCSitesAdminContainer -p 7002:7002 -p 9002:9002 --env-file ./wcsitesserver.env.list oracle/wcsites:12.2.1.3 /bin/bash -c "/u01/oracle/sites-container-scripts/startSitesServer.sh; /bin/bash"
+```
+Managed Container start up command explained:
+```
+   | --name                 | container_name      | Database name; set to ‘WCSitesManagedContainer’                              |
```

Comment:
If you remove the indent from these lines, Markdown would create a proper table. You can then use back ticks inside the table to format it monospaced.